### PR TITLE
quick change back (mixed package.json)

### PR DESCRIPTION
**Pull request by judeamos**
I made changes in the wrong package.json file in this PR (https://github.com/dev-launchers/dev-launchers-platform/pull/1877). I've now undone the changes i made in the wrong file. Again, it is better to review in this PR (https://github.com/dev-launchers/dev-launchers-platform/pull/1875)

### CODE REVIEW

File: apps/user-profile/package.json [hunk 1]
```diff
@@ -78,6 +78,10 @@
     "not ie <= 11",
     "not op_mini all"
   ],
+  "lint-staged": {
+    "*.js": "eslint '*/**/*.{js,jsx}' --fix",
+    "*.{js,css,md}": "prettier --write"
+  },
```
Comment (Enjoy2Live) [inline apps/user-profile/package.json:84]: @judeamos 
this should be deleted, also please don't ignore code-owners alert and always seek their review before merging

Comment (judeamos) [reply apps/user-profile/package.json:84]: I was reverting the change back to the original, as I accidentally changed the one in the universal design folder. So the package.json in the user profile folder already had this. Thought there was no need for review as this was there by default. Where changes are being made outside of our project folder, or any app-level changes I'll make sure to include it in the description, just like I've done here: (https://github.com/dev-launchers/dev-launchers-platform/pull/1875/files)

Comment (judeamos) [reply apps/user-profile/package.json:84]: Do you need me to remove this, even though this was their originally?

Comment (judeamos) [reply apps/user-profile/package.json:84]: Oh i've just checked the commits log, it was, my bad. Reverting it back.

**Pull request merged**